meteor/lib/api/peripheralDevice.ts

```diff
@@ -122,6 +122,13 @@ export interface NewPeripheralDeviceAPI {
 		rundownExternalId: string,
 		ingestSegment: IngestSegment
 	): Promise<void>
+	dataSegmentRanksUpdate(
+		deviceId: PeripheralDeviceId,
+		deviceToken: string,
+		rundownExternalId: string,
```

Comment:
I don't like this api signature having two arrays that need to line up.
It would make more sense to have it as
```
segmentRanks: { [segmentExternalId]: number }
```

Reply:
I've made this change